doc/source/whatsnew/v0.18.0.txt

```diff
@@ -275,6 +276,171 @@ other anchored offsets like ``MonthBegin`` and ``YearBegin``.
    d + pd.offsets.QuarterBegin(n=0, startingMonth=2)
 
 
+.. _whatsnew_0180.breaking.resample:
+
+Resample API
+^^^^^^^^^^^^
+
+Like the change in the window functions API :ref:`above <whatsnew_0180.enhancements.moments>`, ``.resample(...)`` is changing to have
+a more groupy-like API. (:issue:`11732`).
+
+.. ipython:: python
+
+   np.random.seed(1234)
+   df = pd.DataFrame(np.random.rand(10,4),
+                     columns=list('ABCD'),
+                     index=pd.date_range('2010-01-01 09:00:00', periods=10, freq='s'))
+   df
+
+
+**Previous API**:
+
+You would write a resampling operation that immediately evaluates.
+
+This defaults to ``how='mean'``
+
+.. code-block:: python
+
+   In [6]: df.resample('2s')
+   Out[6]:
+                            A         B         C         D
+   2010-01-01 09:00:00  0.485748  0.447351  0.357096  0.793615
+   2010-01-01 09:00:02  0.820801  0.794317  0.364034  0.531096
+   2010-01-01 09:00:04  0.433985  0.314582  0.424104  0.625733
+   2010-01-01 09:00:06  0.624988  0.609738  0.633165  0.612452
+   2010-01-01 09:00:08  0.510470  0.534317  0.573201  0.806949
+
+You could also specify a ``how`` directly
+
+.. code-block:: python
+
+   In [7]: df.resample('2s',how='sum')
+   Out[7]:
+                            A         B         C         D
+   2010-01-01 09:00:00  0.971495  0.894701  0.714192  1.587231
+   2010-01-01 09:00:02  1.641602  1.588635  0.728068  1.062191
+   2010-01-01 09:00:04  0.867969  0.629165  0.848208  1.251465
+   2010-01-01 09:00:06  1.249976  1.219477  1.266330  1.224904
+   2010-01-01 09:00:08  1.020940  1.068634  1.146402  1.613897
+
+.. warning::
```

Comment:
hmm, ok, I will update a bit then. I always like to show, hey here is what you used to write, and here is what the new API will be (rather than the other way around)